### ISSUE TITLE
Warp doesn't work with screen readers on Linux

### ISSUE THREAD

**Issue by paul-geoghegan** [warpdotdev/Warp#8979]
### Pre-submit Checks

- [x] I have [searched Warp bugs](https://github.com/warpdotdev/warp/issues?q=is%3Aissue+label%3ABUG) and there are no duplicates
- [x] I have [searched Warp known issues page](https://docs.warp.dev/help/known-issues) and my issue is not there
- [ ] I have an issue with AI and have included the debugging ID (Optional, but helps expedite the AI quality fix). [Debugging ID instructions](https://docs.warp.dev/agent-platform/agent/ai-faqs#gathering-ai-debugging-id)
- [ ] I have technical issue and have included the logs (optional, but helps expedite the bug fix). [Log instructions](https://docs.warp.dev/support-and-community/troubleshooting-and-support/sending-us-feedback#gathering-warp-logs)

### Describe the bug

i there, I was hoping to try Warp as I seen a lot of people talking about it. I'm blind and use the Orca screen reader on Ubuntu. After opening Warp though it just says "Warp button" and I can't seem to navigate or interact with the app at all.

### To reproduce

1. On Ubuntu press super+alt+s to enable Orca
2. Open the overview pannel and type Warp
3. Press enter to open Warp
4. Notice that you can't interact with Warp.

### Expected behavior

Screen readers should be able to use and interact with the app

### Operating system (OS)

Linux

### Operating system and version

Ubuntu 26.04

### Shell Version

GNU bash, version 5.3.9(1)-release (x86_64-pc-linux-gnu)

### Current Warp version

Latest

### Regression

No, this bug or issue has existed throughout my experience using Warp

### Recent working Warp date

N/A

### Does this block you from using Warp daily?

Yes, this issue prevents me from using Warp daily.

### Is this an issue only in Warp?

Yes, I confirmed that this only happens in Warp, not other terminals.

### Warp Internal (ignore): linear-label:b9d78064-c89e-4973-b153-5178a31ee54e

None
